bundle/regal/lsp/completion/main.rego

```diff
@@ -10,4 +10,9 @@ import rego.v1
 # METADATA
 # description: main entry point for completion suggestions
 # entrypoint: true
-items contains data.regal.lsp.completion.providers[_].items[_]
+items contains item if {
+	some provider
+	completion := data.regal.lsp.completion.providers[provider].items[_]
+
+	item := object.union(completion, {"_regal": {"provider": provider}})
```

Comment:
Only the `rulerefs` provider used to return this before, but knowing which provider is responsible for a suggestion is really helpful for debugging, so I moved this to the main router to get it for all providers.